bacchus_move_base/scripts/send_goals.py

```diff
@@ -32,9 +32,9 @@ def __init__(self):
 
         self.goal_cnt = 0
         #Create action client
-        self.client = actionlib.SimpleActionClient('thorvald_001/move_base',MoveBaseAction)
+        self.client = actionlib.SimpleActionClient('/move_base',MoveBaseAction)
         rospy.loginfo("Waiting for move_base action server...")
-        wait = self.client.wait_for_server(rospy.Duration(5.0))
+        wait = self.client.wait_for_server()
```

Comment:
Does it work without the rospy.Duration(5.0)? This is something I added in the past because it was not working without on my system

Reply:
This looks very ugly and unnecessary… without duration it should work fine. If not, something else is wrong.

Reply:
it works fine for me without the duration